relay-server/src/actors/project_cache.rs

```diff
@@ -37,7 +40,7 @@ impl ResponseError for ProjectError {}
 
 struct ProjectEntry {
     last_updated_at: Instant,
-    project: Addr<Project>,
+    project: Project,
```

Comment:
You can move `last_updated_at` into `Project` now and remove the `ProjectEntry` type. The only reason we did not do this initially was because it would have required async messages to read this value. Since we can now simply access this via a sync getter function, it is best placed and managed within the project. The `state` or `state_clone` function could automatically update the update timer.

Reply:
ok